Makefile

```diff
@@ -1,3 +1,4 @@
+SHELL:=/bin/bash
```

Comment:
Can you describe what problem has been solved?

Reply:
some environments link sh -> dash

Reply:
some environments link sh -> dash